client/src/components/EditReviewForm.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import ErrorList from "./layout/ErrorList.js";
+
+const EditReviewForm = (props) => {
+  const [editReview, setEditReview] = useState({
+    title: props.title,
+    content: props.content,
+    rating: props.rating,
+  });
+
+  const handleInputChange = (event) => {
+    setEditReview({
+      ...editReview,
+      [event.currentTarget.name]: event.currentTarget.value,
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    props.patchReview(editReview, props.id, props.toggleEdit);
```

Comment:
okay, this is rather clever passing toggleEdit in here, but the downside is it creates some weird coupling in your app. 

This is what I would do
- have `patchReview` return true or false
- Note: this means you may have to await `props.patchReview`, which means youll also have to make your handleSbumit async
- below line 20 in this component, if patchReview returns true, you'll toggleEdit, if not, you wont run toggle edit